ui/components/app/confirm/info/row/currency.tsx

```diff
@@ -21,6 +22,7 @@ export type ConfirmInfoRowCurrencyProps = {
 export const ConfirmInfoRowCurrency = ({
   value,
   currency,
+  dataTestId,
```

Comment:
Sorry missing the obvious, also a very minor one. 
Would it also work if we add this `dataTestId` straight to the `Box` component instead of adding both `CurrencyDisplay` and `UserPreferencedCurrencyDisplay` and read the text?